### ISSUE TITLE
`reset_unmatched_scores` not respected in V5

### ISSUE THREAD

**Issue by yammes08** [recyclarr/recyclarr#187]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

Recyclarr V5 is appears to be resetting the unmatched scores (to 0) regardless of that setting

### Expected Behavior

Scores are never altered unless it is listed in the trash_ids array and has a valid score to assign (either from the guide or via an explicit score).

### Steps To Reproduce

1. Upgrade to V5
2. Confirm correct configuration of `reset_unmatched_scores`, e.g.:
```
quality_profiles:
    - name: WEB-1080p
        reset_unmatched_scores: false
```
3. Add a score for a non-TRaSH guide CF to that quality profile
4. Run `recyclarr sync`

### Environment & Versions

```bash
- Operating System:
- Architecture:
- Recyclarr Version:
- Recyclarr Build:
- Sonarr Version:
- Radarr Version:
All TBC
```

### Recyclarr Log File(s)

TBC

### Configuration YAML

```YAML
TBC
```

**Comment by kanine** [comment]
[WEB-1080p Bronze.zip](https://github.com/recyclarr/recyclarr/files/11847646/WEB-1080p.Bronze.zip)

Example yaml file that I had an issue with the Profile Name is "WEB-1080p Bronze", which is a direct clone of the Trash Guide WEB-1080p that I adjust with some additional CF Scores outside of the standard guide ones.

**Comment by yammes08** [comment]
> [WEB-1080p Bronze.zip](https://github.com/recyclarr/recyclarr/files/11847646/WEB-1080p.Bronze.zip)
> 
> 
> 
> Example yaml file that I had an issue with. The Profile Name is "WEB-1080p Bronze", which is a direct clone of the TRaSH Guide WEB-1080p that I adjust with some additional CF Scores outside of the standard guide ones.

Can you upload the contents to pastebin instead? Much better than an unknown zip file 😉

**Comment by rcdailey** [comment]
Resolved in v5.0.1.
